LEGO1/lego/legoomni/include/legocameracontroller.h

```diff
@@ -1,17 +1,21 @@
 #ifndef LEGOCAMERACONTROLLER_H
 #define LEGOCAMERACONTROLLER_H
 
+#include "legopointofviewcontroller.h"
 #include "mxcore.h"
+#include "mxpoint32.h"
 #include "realtime/matrix.h"
 #include "realtime/vector.h"
 
 // VTABLE: LEGO1 0x100d57b0
 // SIZE 0xc8
-class LegoCameraController : public MxCore {
+class LegoCameraController : public LegoPointOfViewController {
```

Comment:
Is it called LegoPointOfViewController in the source or just LegoPOVController

Reply:
LegoPointOfViewController